notebooks/contributing/formatting.md

```diff
@@ -0,0 +1,289 @@
+
+```
+**Learning Objectives:**
+
+- How geophysical inversion is carried out using SimPEG.
+- How to assign appropriate uncertainties to gravity anomaly data.
+- How to design a suitable mesh for gravity inversion when using the integral formulation.
+- How to choose and set parameters for the inversion.
+- How to define directives that are applied and updated throughout the inversion.
+- How to applying the sensitivity weighting generally used in 3D gravity inversion.
+- How to invert data using weighted least-squares and sparse-norm regularization.
+- How to analyse inversion results.
+```
+
+## Import Functionality
+
+Following the introduction section, the contributor must import all functionality used by the tutorial and briefly explain the functionality that is most important.
+
+### Cell 1: Header and Summary
+
+This Markdown cell uses the heading **Import Modules**. For functionality that relates
+directly to the purpose of the tutorial, some context and a link to API documentation
+should be provided. E.g. 
+
+:::
+## Import Modules
+
+Here, we import all of the functionality required to run the notebook for the tutorial
+exercise. All of the functionality specific to simulating gravity data are imported from
+[simpeg.potential_fields.gravity](xref:simpeg#simpeg.potential_fields.gravity). We also
+import some useful utility functions from [simpeg.utils](xref:simpeg#simpeg.utils).
+To simulate gravity data, we need to define our problem geometry on a numerical grid
+(or mesh). To generate the mesh, we used the
+[discretize](https://discretize.simpeg.xyz/en/main) package.
+:::
+
+### Cell 2: Code Cell
+
+The coding cell where functionality is imported should be organized. E.g.
+
+```python
+# SimPEG functionality
+from simpeg.potential_fields import gravity
+from simpeg.utils import plot2Ddata, model_builder
+from simpeg import maps
+
+# discretize functionality
+from discretize import TensorMesh
+from discretize.utils import mkvc, active_from_xyz
+
+# Common Python functionality
+import numpy as np
+from scipy.interpolate import LinearNDInterpolator
+import matplotlib as mpl
+
+mpl.rcParams.update({"font.size": 14})
+import matplotlib.pyplot as plt
+import os
+```
+
+## Tutorial Sections (and Subsections)
+
+Each section (or subsection) is constructed using a combination of Markdown and code cells. The structure and formatting of these are discussed below.
+
+### Header and Summary
+
+Each section (or subsection) begins with a Markdown cell containing the header and a summary.
+Here, the contributor must:
+
+* summarize what is being done in the section
+* describe any new functionality that is being introduced
+* provide links to all relevant API documentation
+* describe the choices for hyperparameter values used in the tutorial
+
+E.g.
+
+```
+## Define the Survey
+
+Surveys within SimPEG generally require the user to create and connect three
+types of objects:
```

Comment:
```suggestion
```markdown
## Define the Survey

Surveys within SimPEG generally require the user to create and connect three
types of objects:
```